pkg/scheduler/framework/plugins/noderesources/requested_to_capacity_ratio.go

```diff
@@ -61,18 +60,18 @@ func NewRequestedToCapacityRatio(plArgs *runtime.Unknown, handle framework.Frame
 	if err := framework.DecodeInto(plArgs, args); err != nil {
 		return nil, err
 	}
-	p := &ResourceAllocationPriority{RequestedToCapacityRatioName, buildRequestedToCapacityRatioScorerFunction(args.FunctionShape, args.ResourceToWeightMap), args.ResourceToWeightMap}
+
 	return &RequestedToCapacityRatio{
-		handle:   handle,
-		priority: p,
+		handle:                   handle,
+		resourceAllocationScorer: resourceAllocationScorer{RequestedToCapacityRatioName, buildRequestedToCapacityRatioScorerFunction(args.FunctionShape, args.ResourceToWeightMap), args.ResourceToWeightMap},
```

Comment:
nit, please break this into multiple lines so that it is easier to read:

```
resourceAllocationScorer: resourceAllocationScorer{
           RequestedToCapacityRatioName, 
           buildRequestedToCapacityRatioScorerFunction(args.FunctionShape, args.ResourceToWeightMap), 
           args.ResourceToWeightMap,
},
```